src/main/java/org/jenkinsci/plugins/github/webhook/subscriber/DefaultPushGHEventSubscriber.java

```diff
@@ -76,7 +76,22 @@ protected void onEvent(final GHSubscriberEvent event) {
         URL repoUrl = push.getRepository().getUrl();
         final String pusherName = push.getPusher().getName();
         LOGGER.info("Received PushEvent for {} from {}", repoUrl, event.getOrigin());
-        final GitHubRepositoryName changedRepository = GitHubRepositoryName.create(repoUrl.toExternalForm());
+        GitHubRepositoryName fromEventRepository = GitHubRepositoryName.create(repoUrl.toExternalForm());
+
+        if (fromEventRepository == null) {
+            // On push event on github.com url === html_url
+            // this is not consistent with the API docs and with hosted repositories
+            // see https://goo.gl/c1qmY7
+            // let's retry with 'html_url'
+            URL htmlUrl = push.getRepository().getHtmlUrl();
+            fromEventRepository = GitHubRepositoryName.create(htmlUrl.toExternalForm());
+            if (fromEventRepository != null) {
+                LOGGER.info("PushEvent handling: 'html_url' field "
```

Comment:
info is printed by default in jenkins logger, this will lead to redundant messages
Please change to trace or debug

Reply:
done in c0c83fb commit.